test/dotnet-compile.Tests/CompilerTests.cs

```diff
@@ -228,6 +231,42 @@ public void CanSetOutputAssemblyNameForApps()
                     "MyApp.runtimeconfig.json", "MyApp.deps.json" });
         }
 
+        [Fact]
+        public void CompiledOutput_WithServiceableAttribute_CreateServiceableBinary()
+        {
+            const string appName = "EndToEndTestApp";
+            var testInstance = TestAssetsManager.CreateTestInstance(appName).WithLockFiles();
+            var root = testInstance.TestRoot;
+
+            // run compile
+            var outputDir = Path.Combine(root, "bin");
+            var testProject = ProjectUtils.GetProjectJson(root, appName);
+            var buildCommand = new BuildCommand(testProject, output: outputDir, framework: DefaultFramework);
+            var result = buildCommand.ExecuteWithCapturedOutput("--Serviceable");
+            result.Should().Pass();
+
+            bool found = false;
+            string resultFile = Path.Combine(outputDir, "Debug", "netcoreapp1.0", buildCommand.GetOutputExecutableName());
+            using (FileStream fs = new FileStream(resultFile, FileMode.Open, FileAccess.Read))
+            {
+                MetadataReader reader = MetadataReaderProvider.FromMetadataStream(fs).GetMetadataReader();
+                foreach (CustomAttributeHandle cah in reader.CustomAttributes)
+                {
+                    if (cah.As<AssemblyMetadataAttribute>() != null)
+                    {
+                        AssemblyMetadataAttribute a = cah.As<AssemblyMetadataAttribute>();
+                        if (a.Key == "Serviceable")
+                        {
+                            found = true;
+                            Assert.True(a.Value.ToLower() == "true");
```

Comment:
Please, keep with the fluent assertion style. a.Value.ToLower().Should().Be("true");